src/main/java/BrainRot/Storage.java

```diff
@@ -0,0 +1,84 @@
+package BrainRot;
+
+import java.io.*;
```

Comment:
It may be better to specify the classes individually instead of using wildcard imports.